fixes/consolidate-multiple-richtext-history.sql

```diff
@@ -99,37 +99,37 @@ declare @q6 varchar(max)
 select @q6 = ';
 with H as (
 	select ID, AuditBegin, AuditEnd, R=ROW_NUMBER() over(partition by ID order by AuditBegin)
-	from dbo.' + @TableName + '
+	from dbo.' + @Table + '
 )
-update dbo.' + @TableName + ' set AuditEnd=B.AuditBegin
+update dbo.' + @Table + ' set AuditEnd=B.AuditBegin
 from H A
 left join H B on A.ID=B.ID and A.R+1=B.R
-where ' + @TableName + '.ID=A.ID and ' + @TableName + '.AuditBegin=A.AuditBegin
+where ' + @Table + '.ID=A.ID and ' + @Table + '.AuditBegin=A.AuditBegin
```

Comment:
Should we qualify table name here?

Reply:
Schema-qualify, as in `dbo.[Table]`? I don't think so...